sys/shell/commands/sc_icmpv6_echo.c

```diff
@@ -36,7 +36,7 @@ static uint16_t min_seq_expected = 0;
 static uint16_t max_seq_expected = 0;
 static char ipv6_str[IPV6_ADDR_MAX_STR_LEN];
 
-static void usage(char **argv)
+static void usage(char * *argv)
```

Comment:
As stated in the desciption.
> Btw. I used uncrustify the first time and it did some things of which I don't know if they're intentional.

So, this is no new kind of coding style for pointer types?
Shall I manually correct it and write an issue regarding `uncrustify-riot.cfg`?